### PR TITLE
Reformatting Operations

### PR DESCRIPTION
**Pull request by GenericP3rson**
Reformatted operations to be in separate files. Will be happy to workshop how much or little separation we want in the files—right now, it's slightly arbitrary but should be more or less consistent. Once this format is decided, I'll edit the functionals file with similar updates.

Other misc. updates from this PR:

- CCX was duplicated, so chose the class with more information
- Converted some eigenvalue vectors from numpy arrays to torch tensors
- Added the U gate alias (same as the U3)

### CODE REVIEW

File: torchquantum/operator/op_hamil_exp.py
Comment (GenericP3rson) [inline torchquantum/operator/op_hamil_exp.py:1]: (There are no actual updates to this file; I ran the formatter on the folder which appears to have updated it.)

**Comment by GenericP3rson** [comment]
It got a bit messy with the test reformatting, but here are some major updates:

- All operations are in the `standard_gates` folder (they should still be useable as before)
- Automatic creation of the dictionary (since each class has an extra "op_name" attribute)
- Automatic creation of the __all__ list

File: torchquantum/operator/standard_gates/__init__.py
Comment (GenericP3rson) [inline torchquantum/operator/standard_gates/__init__.py:1]: This file primarily has the core of the new updates (creation of op_name_dict, specific imports, etc.)

**Review by Hanrui-Wang — APPROVED** [review]
reviewed

**Pull request merged**